examples/stocks/lib/main.dart

```diff
@@ -94,6 +94,13 @@ class StocksAppState extends State<StocksApp> {
     return null;
   }
 
+  Future<LocaleQueryData> _onLocaleChanged(ui.Locale locale) {
+    return initializeMessages(locale.toString()).then((_) {
+      Intl.defaultLocale = locale.toString();
```

Comment:
Should we avoid calling locale.toString twice by caching the result in a variable?